### PR TITLE
Replace strict type var naming convention

### PR DESCRIPTION
**Pull request by SimonHeybrock**
Not entirely happy with this either. Should it simply be removed entirely, including the examples in the rule above? Or are we still happy with `RunType`?

### CODE REVIEW

**Comment by jl-wynen** [comment]
This reads like a general advice but not a hard guideline. So in the interest of making the guidelines enforceable, I'd say remove it.

This advice / guide now reminds me of some naming conventions that encode the kind of a variable in the name. E.g., https://root.cern/TaligentDocs/TaligentOnline/DocumentRoot/1.0/Docs/books/WM/WM_63.html#HEADING77 Personally, I don't like those. We should apply good judgement when picking names.

**Comment by SimonHeybrock** [comment]
Ok, removed. But what about the example for `RunType` and `MonitorType` in the section before? Are we still happy with those recommendations?

**Review by jl-wynen — COMMENTED** [review]
Can you briefly document the reason for removal? Just so we don't need to dig too much in the future.

**Comment by SimonHeybrock** [comment]
> Can you briefly document the reason for removal? Just so we don't need to dig too much in the future.

Can you suggest something?

File: docs/user-guide/reduction-workflow-guidelines.md [hunk 1]
```diff
@@ -92,13 +92,7 @@ RunType = TypeVar('RunType', SampleRun, BackgroundRun)
...
 ```
 
-### C.3: Use the suffix 'Type' for type vars
-
-**Reason**
-This makes it easier to distinguish type vars from concrete domain types.
-
-**Example**
-See 'RunType' and 'MonitorType' in the table of C.2.
+### C.3: (Removed rule on naming TypeVars)
```
Comment (jl-wynen) [inline docs/user-guide/reduction-workflow-guidelines.md:95]:
```suggestion
### C.3: (Removed rule on naming TypeVars)

This guideline was too restrictive as not all TypeVars represent a "type", conceptually.
Instead, authors should apply good judgment when naming TypeVars.
```

**Review by jl-wynen — APPROVED** [review]

**Pull request merged**